src/utf8_string.cpp

```diff
@@ -94,5 +96,7 @@ namespace Sass {
       return utf16;
     }
 
+    #endif
+
```

Comment:
@mgreter, I kept MinGW and MSVC implementations separate here: https://github.com/sass/libsass/blob/e6ac0a2/src/file.cpp#L28-L44 (as GCC hasn't implemented C++11 `<codecvt>`, so MinGW implementation should make a win32 API call for conversion instead).
IMO, if this solution works for both, we should probably use the same there. Either way, we should probably refactor this out at one place in `util.cpp` for DRY.